cli/tests/integrations/test_package.py

```diff
@@ -360,14 +321,6 @@ def test_package_metadata():
     for label, value in expected_labels.items():
         assert value == six.b(app_labels.get(label))
 
-    # these labels are different for cosmos b/c of null problem
-    # we have cosmos tests for test, and will fix in issue 431
-    if os.environ.get(constants.COSMOS_URL_ENV) is None:
-        assert expected_metadata == six.b(
-            app_labels.get('DCOS_PACKAGE_METADATA'))
-        assert expected_command == six.b(
-            app_labels.get('DCOS_PACKAGE_COMMAND'))
```

Comment:
Let's not remove this completely. Let's make sure that we add this back when we fix nulls in cosmos.